src/hooks/useUserListFilterUrlParams.ts

```diff
@@ -0,0 +1,38 @@
+import { useCallback } from 'react';
+import { useRouter } from 'next/router';
+import { useUrlParams } from '@taskany/bricks';
+import { debounce } from 'throttle-debounce';
+
+import { UserFilterQuery } from '../modules/userTypes';
+
+export const useUserListFilterUrlParams = () => {
+    const router = useRouter();
+    const pushUrl = useCallback((url: string) => router.push(url), [router]);
+
+    const { values, setter, clearParams } = useUrlParams(
+        {
+            search: 'string',
```

Comment:
Let's rename `getUserListSchema` fields by removing `Query` from them. This will make our urls look better